microsoft-edge/devtools-guide-chromium/about-tools.md

```diff
@@ -5,25 +5,27 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 02/02/2022
+ms.date: 06/21/2023
 ---
 # About the list of tools
 
 DevTools provides 35 tools:
```

Comment:
experiments can add more tool tabs.  extensions can add more tool tabs.

Reply:
Because experiments and extensions can add tabs, and because things change often, wouldn't it be better to say something like "DevTools provides more than 30 tools", instead of being specific?